stan/math/prim/prob/std_normal_ccdf_log.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef STAN_MATH_PRIM_PROB_STD_NORMAL_CCDF_LOG_HPP
+#define STAN_MATH_PRIM_PROB_STD_NORMAL_CCDF_LOG_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/prob/std_normal_lccdf.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup prob_dists
+ * @deprecated use <code>std_normal_lccdf</code>
+ */
+template <typename T_y>
+inline return_type_t<T_y> std_normal_ccdf_log(const T_y& y) {
```

Comment:
I asked the same question in the description of the PR... :) I've added them only because I was not sure if Stan would expect those, or if we are making promises as to how functions can be called by users... I'll be very happy to remove them from this PR if we could. @bob-carpenter?

Reply:
I had a look at this and we should add the deprecated stuff.

When dealing with truncation, things like:

```
x ~ normal(y, 1) T[, 1];
```

The old compiler still references the *cdf_log functions.

The new compiler has a bug it looks like cause it's not generating functions at all: https://github.com/stan-dev/stanc3/issues/414

I guess since the deprecated-ness isn't resolved upstream, we leave these in.